.github/PROCESS.md

```diff
@@ -10,6 +10,32 @@ We publish new issues bi-weekly, **every other Friday at 10:00 AM**. This includ
 
 We hold a **weekly Tuesday meeting at 2:00 PM** to check-in, collaborate, discuss updates, make decisions, etc.
 
+## :snowflake: Roles
+
+There are a few different areas of responsibility for each issue. Who does what is determined during weekly meetings, over emails, and chat. Some responsibilities are **collective**. :busts_in_silhouette:
+
+#### Lead and Co-lead
```

Comment:
I'd be fine removing these two roles altogether, and replace this with a "person responsible for hitting the publish button" (i.e. github PR merge).

Reply:
I'm fine with that.

How do you think we should word the rest of the doc that refers to lead/co-lead?

Reply:
Maybe we make up a "publisher" role (or a better word, if someone can think of one), which is the person who "hits the publish button" (i.e. merges the PR), and refer to that "publisher" instead of lead/co-lead.